natural-language-understanding/v1.ts

```diff
@@ -20,6 +20,7 @@ import GeneratedNaturalLanguageUnderstandingV1 = require('./v1-generated');
 class NaturalLanguageUnderstandingV1 extends GeneratedNaturalLanguageUnderstandingV1 {
   static VERSION_DATE_2016_01_23: string = '2016-01-23';
   static VERSION_DATE_2017_02_27: string = '2017-02-27';
+  static VERSION_DATE_2018_04_05: string = '2018-04-05';
```

Comment:
Please don't add the version date as constant. We decided not to use constants anymore to enforce users to type the date.

Reply:
Sounds good. I'm confirming if we're removing the constants, does this mean this PR will need to change to pass in the string '2018-04-05' directly instead? https://github.com/watson-developer-cloud/natural-language-understanding-nodejs/pull/23

Reply:
@aluu317 Yes! Please change that to just pass in that string.